cdap-ui/app/cdap/components/PipelineDetails/RunLevelInfo/RunConfigs.js

```diff
@@ -84,8 +84,12 @@ export default class RunConfigs extends Component {
 
     let runtimeArgs = objectQuery(this.props.currentRun, 'properties', 'runtimeArgs') || '';
     try {
-      runtimeArgs = JSON.parse(runtimeArgs);
-      delete runtimeArgs[''];
+      if (runtimeArgs === '') {
+        runtimeArgs = {};
+      } else {
+        runtimeArgs = JSON.parse(runtimeArgs);
+        delete runtimeArgs[''];
```

Comment:
why does runtimeArgs have a key that is empty string? that seems like a bug

Reply:
Not sure why, this has been there for over 2 years. @ajainarayanan has more context? The change I made here is just to suppress unnecessary `ERROR: cannot parse runtime arguments` log in console when we set `runtimeArgs` to be an empty string in line 85.